site/src/pages/docs/client-retry.mdx

```diff
@@ -246,8 +249,11 @@ Second, it occurs when the time of individual attempt in retry has passed the ti
 You can configure it when you create the decorator:
 
 ```java
-RetryingClient.newDecorator(rule, maxTotalAttempts,
-                            responseTimeoutMillisForEachAttempt);
```

Comment:
The deprecated spec is as follows.
https://javadoc.io/static/com.linecorp.armeria/armeria-javadoc/1.32.3/com/linecorp/armeria/client/retry/RetryingClient.html#newDecorator(com.linecorp.armeria.client.retry.RetryRule,int,long)